src/views/UserExports/index.js

```diff
@@ -1,383 +1,313 @@
 import FormattedDate from '#rscv/FormattedDate';
+import RawTable from '#rscv/RawTable';
+import TableHeader from '#rscv/TableHeader';
 import LoadingAnimation from '#rscv/LoadingAnimation';
 
 import BackLink from '#components/general/BackLink';
 import {
-    createUrlForExport,
-    createParamsForGet,
-    createUrlForExportsOfProject,
-
-    transformResponseErrorToFormError,
-} from '#rest';
-import {
-    userExportsListSelector,
-    setUserExportsAction,
-    setUserExportAction,
-
     projectIdFromRouteSelector,
 } from '#redux';
 import { pathNames } from '#constants';
 import { mimeTypeToIconMap } from '#entities/lead';
-import schema from '#schema';
 import notify from '#notify';
 import _ts from '#ts';
 import ExportPreview from '#components/other/ExportPreview';
+import useRequest from '#utils/request';
 
 import styles from './styles.scss';
 
+const emptyList = [];
+
 const propTypes = {
-    userExports: PropTypes.array.isRequired, //eslint-disable-line
-    setUserExports: PropTypes.func.isRequired,
-    setUserExport: PropTypes.func.isRequired,
     projectId: PropTypes.number,
 };
 
 const defaultProps = {
-    userExports: [],
     projectId: undefined,
 };
 
 const mapStateToProps = state => ({
-    userExports: userExportsListSelector(state),
     projectId: projectIdFromRouteSelector(state),
 });
 
-const mapDispatchToProps = dispatch => ({
-    setUserExports: params => dispatch(setUserExportsAction(params)),
-    setUserExport: params => dispatch(setUserExportAction(params)),
-});
-
-const emptyList = [];
+const tableKeyExtractor = d => d.id;
 
-@connect(mapStateToProps, mapDispatchToProps)
-export default class UserExports extends React.PureComponent {
-    static propTypes = propTypes;
-    static defaultProps = defaultProps;
-    static tableKeyExtractor = d => d.id;
+const maxItemsPerPage = 25;
 
-    constructor(props) {
-        super(props);
+function UserExports(props) {
+    const { projectId } = props;
 
-        this.state = {
-            selectedExport: 0,
-            pendingExports: true,
-        };
+    const [userExports, setUserExports] = useState();
+    const [selectedExport, setSelectedExport] = useState();
+    const [exportCount, setExportCount] = useState(0);
+    const [activeSort, setActiveSort] = useState('-exported_at');
+    const [activePage, setActivePage] = useState(1);
+    const [
+        pending,
+        ,
+        ,
+    ] = useRequest({
+        url: 'server://exports/',
+        query: {
+            project: projectId,
+            ordering: activeSort,
+            is_preview: false,
+            offset: (activePage - 1) * maxItemsPerPage,
+            limit: maxItemsPerPage,
+        },
+        method: 'GET',
+        autoTrigger: true,
+        shouldPoll: (response) => {
+            const isPending = response?.results.some(e => e.status === 'pending' || e.status === 'started');
+            if (isPending) {
+                return 5000;
+            }
```

Comment:
why return 5000?

Reply:
Power level more than 5000!!
Its the polling time